src/big_int.rs

```diff
@@ -211,7 +211,6 @@ impl BigInt {
     }
 
     /// Divide `self` by the given `BigInt` exactly if this is possible. Return
```

Comment:
Fix comment, something like this?:
```suggestion
    /// Divide `self` by the given `BigInt` exactly if this is possible.
    /// Return an error if `other` is zero or if `self` not divisible by other.
```